pantheon-bundle/frontend/src/app/versions.tsx

```diff
@@ -744,6 +747,18 @@ class Versions extends Component<IProps, IState> {
                 })
         }
     }
+
+    private capitalize = (str) => {
+        if (str === undefined || str.trim().length === 0) {
+            return
+        }
+        return str.charAt(0).toUpperCase() + str.slice(1)
+    }
+
+    private getAlertTitle = () => {
```

Comment:
Could I please ask that this method be renamed? I would expect that a method named "getXXX" is a "safe"/read-only operation, but inside of it you are calling a setState. Maybe "resetAlertTitle" or "determineAlertTitle" or something else if you have a better idea.

Reply:
updated